tests/run-custom-args/run-macros-erased/reflect-isFunctionType/macro_1.scala

```diff
@@ -1,30 +1,30 @@
 import scala.quoted._
 
 
-inline def isFunctionType[T:Type]: Boolean = ${ isFunctionTypeImpl('[T]) }
+inline def isFunctionType[T]: Boolean = ${ isFunctionTypeImpl('[T]) }
```

Comment:
This is nice, improves meta-programming experience a lot 👍

Reply:
This proves the point made in the previous comment. Here the calls failed to compile and I noticed at once that we had an extra `Type` that was not needed.